src/test/fixtures/exchange/mockxchange.js

```diff
@@ -36,6 +36,11 @@ export const mockxchange = resolvers => {
         }
       },
     },
+    Order: {
+      __resolveType(obj, _context, _info) {
+        return "BuyOrder"
+      },
+    },
```

Comment:
These are so that tests can know what an underlaying concrete type is  when asking for an "Order" - it will need to change in the future when there are test for offers